### PR TITLE
 python3.pkgs.cherrypy: 18.1.0 -> 18.1.1

### PR DESCRIPTION
**Pull request by dotlambda**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/cherrypy/default.nix [hunk 1]
```diff
@@ -1,54 +1,39 @@
 { lib, buildPythonPackage, fetchPypi, isPy3k
```
Comment (worldofpeace) [inline pkgs/development/python-modules/cherrypy/default.nix]:
```suggestion
{ lib, buildPythonPackage, fetchPypi
```

**Review by worldofpeace — CHANGES_REQUESTED** [review]
Can you change `pname` to **c**herry**p**y?

File: pkgs/development/python-modules/cherrypy/default.nix (hunk 1 above)
Comment (dotlambda) [reply pkgs/development/python-modules/cherrypy/default.nix]: I added `disabled = !isPy3k` just to make sure.

**Review by worldofpeace — APPROVED** [review]
Built locally for all interpreters.

**Pull request merged**